_posts/2018-05-09-wpe-2.21.1-released.md

```diff
@@ -3,8 +3,7 @@ layout: post
 title: "WPE WebKit 2.21.1 released!"
 version: 2.21.1
 date: 2018-05-09
-tags: release
-category: unstable
+tags: [release, unstable]
```

Comment:
You could just use: `tags: release unstable`
But I guess the array format works too. Feel free to select the one that is more comfortable for you.

Reply:
According to [the documentation](https://jekyllrb.com/docs/front-matter/) the front matter is YAML and I did not expect that Jekyll would do any extra processing on the input… Personally I prefer to have well-formed YAML using an array instead of relying on (undocumented?) Jekyll's behaviour :wink:

Reply:
No problem on my side, just in case you want something "simpler" :wink: